src/generateFromYarnLock.js

```diff
@@ -97,6 +97,40 @@ export async function resolvePathsAndConflicts(
   return resolvedDeps;
 }
 
+function processResolutionName(res) {
+  if(res.includes('node_modules')) return res.split('/node_modules/')[1].split('/')[0];
+  return res;
+ }
+
+export function applyOverrides(imports, packageJson) {
+  const importmap = imports;
+  if(typeof packageJson.importmap !== 'undefined') {
+    const { overrides } = packageJson.importmap;
+
+    Object.keys(overrides).forEach(dep => {
+      if(Array.isArray(overrides[dep])) {
+        /**
+         * we need to delete first, if someone has kv-storage-polyfill in their dependencies, they
+         * should have "std:kv-storage" in their importmap, not "kv-storage-polyfill"
+         */
+        overrides[dep].forEach(res => {
+          const resolution = processResolutionName(res);
+
+          delete importmap[resolution];
+          delete importmap[`${resolution}/`];
+        });
+        importmap[dep] = overrides[dep];
+      } else {
+        delete importmap[dep];
+        delete importmap[`${dep}/`];
+
+        importmap[dep] = overrides[dep];
+      }
```

Comment:
I think there definitely needs to be a dedicated test file doing unit tests only for `applyOverrides` => e.g. probably best to have 2 dedicated files => I would call it postProcessImportMapp
```js
src/postProcessImportMap.js
test/postProcessImportMap.test.js
```